cli/src/main/java/com/devonfw/tools/ide/tool/aws/Aws.java

```diff
@@ -0,0 +1,99 @@
+package com.devonfw.tools.ide.tool.aws;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.attribute.PosixFilePermission;
+import java.util.Set;
+
+import com.devonfw.tools.ide.context.IdeContext;
+import com.devonfw.tools.ide.environment.EnvironmentVariables;
+import com.devonfw.tools.ide.environment.EnvironmentVariablesType;
+import com.devonfw.tools.ide.process.ProcessContext;
+import com.devonfw.tools.ide.tool.LocalToolCommandlet;
+
+/**
+ * {@link LocalToolCommandlet} for AWS CLI (aws).
+ *
+ * @see <a href="https://docs.aws.amazon.com/cli/">AWS CLI homepage</a>
+ */
+
+public class Aws extends LocalToolCommandlet {
+
+  /**
+   * The constructor.
+   *
+   * @param context the {@link IdeContext}.
+   */
+  public Aws(IdeContext context) {
+
+    super(context, "aws", Set.of(TAG_CLOUD));
```

Comment:
```suggestion
    super(context, "aws", Set.of(Tag.CLOUD));
```

Build failed after merge:
```
[ERROR] /home/runner/work/IDEasy/IDEasy/cli/src/main/java/com/devonfw/tools/ide/tool/aws/Aws.java:[30,33] error: cannot find symbol
  symbol:   variable TAG_CLOUD
  location: class Aws
```